testsuite/cluster-test/src/aws.rs

```diff
@@ -22,9 +22,13 @@ pub struct Aws {
 }
 
 impl Aws {
-    pub fn new() -> Self {
+    pub fn new(k8s: bool) -> Self {
         let ec2 = Ec2Client::new(Region::UsWest2);
-        let workspace = discover_workspace(&ec2);
+        let workspace = if k8s {
```

Comment:
There is still workspace name even if it runs with k8s right? Why put 'k8s' string there?

Reply:
We never use this Aws struct or its workspace with the kubernetes setup, so I have added a static string here.
 
With the kubernetes setup, the workspace is encapsulated by `ClusterSwarmKube`. workspace only affects the grafana baseurl and that is provided by the `get_grafana_baseurl` function